security/PracticalSecurity.md

```diff
@@ -0,0 +1,2 @@
+For this portion of the sprint I installed KeePass as well as Virtual Box.
+I searched and read up on Kali Linux's offensive security courses and found what I needed to know to start the free course. Due to time constraints I did not start the actual course but will do it in the near future.
```

Comment:
I've got Kali Linux installed on my PC as a second boot, it's pretty neat but half of the tools on there are head scratchers. I should look into this course!